tools/changelog-helper.sh

```diff
@@ -100,7 +107,7 @@ group_entries() {
     local index=0
     for category in "${category_order[@]}"; do
         changelog=$(sed -re 's/^(- '"${category}"')/'"${index}"' \1/' <<<"${changelog}")
-        index=$(($index+1))
+        index=$((index+1))
```

Comment:
Not 100% into bash syntax, but is it valid to omit the $

Reply:
`$(( ... ))` is numeric expression evaluation.  Names are variable names.  Strings aren't valid numeric expressions. I'd like spaces after the opening brackets and before the closing ones and around the operator...

Reply:
> I'd like to postpone the style-discussions for now as we will have to discuss/check that when adding shfmt anyway.

A lot of this _is_ style change...

Reply:
> I'd like spaces after the opening brackets and before the closing ones and around the operator...

FYI, `shfmt` enforces spaces around the operator, but not around the brackets. I don't see an option to influence that either. Personally, I'd be ok with this though.